.github/workflows/release-nightly.yml

```diff
@@ -9,4 +9,5 @@ jobs:
     uses: ./.github/workflows/jreleaser.yml
     with:
       release-script-to-run: chore/release-nightly.sh
+      attest-artifacts: false
```

Comment:
```suggestion
```

Reply:
@RafDevX @ludvigch could you apply the commit? Martin suggests to have it for nightly build as well. You could either accept this suggestion or just remove `attest-artifacts` as one of the inputs since it it is always `true`.

Reply:
We removed the input entirely in ab3d1972d8aaa028716ea2738d0a034d084944bb so as to reduce the code's maintenance surface

Reply:
Sounds good!